### ISSUE TITLE
Fix crash in objc_release

### ISSUE THREAD

**Issue by numist** [numist/Switch#115]
### Reason

No reason found. Full stack trace includes libobjc.A.dylib, libsystem_blocks.dylib, libdispatch.dylib, CoreFoundation, HIToolbox, AppKit, libdyld.dylib.
### Link to HockeyApp
- [https://rink.hockeyapp.net/manage/apps/62459/crash_reasons/21172812](https://rink.hockeyapp.net/manage/apps/62459/crash_reasons/21172812)

**Comment by numist** [comment]
This stack isn't very helpful:

```
1  libobjc.A.dylib        objc_release + 15
2  libsystem_blocks.dylib _Block_release + 196
3  libdispatch.dylib      _dispatch_client_callout + 8
4  libdispatch.dylib      _dispatch_main_queue_callback_4CF + 861
5  CoreFoundation         __CFRUNLOOP_IS_SERVICING_THE_MAIN_DISPATCH_QUEUE__ + 9
6  CoreFoundation         __CFRunLoopRun + 2159
7  CoreFoundation         CFRunLoopRunSpecific + 296
8  HIToolbox              RunCurrentEventLoopInMode + 235
9  HIToolbox              ReceiveNextEventCommon + 431
10 HIToolbox              _BlockUntilNextEventMatchingListInModeWithFilter + 71
11 AppKit                 _DPSNextEvent + 964
12 AppKit                 -[NSApplication nextEventMatchingMask:untilDate:inMode:dequeue:] + 194
13 AppKit                 -[NSApplication run] + 594
14 AppKit                 NSApplicationMain + 1832
15 libdyld.dylib          start + 1
```

**Comment by numist** [comment]
Seems very possible that this is the manifestation of the event tap bug mentioned (and fixed!) in #139.

But even if not, the elimination of all Objective-C app code probably did away with this.
